gnomad_constraint/constraint/constraint_pipeline.py

```diff
@@ -58,8 +102,21 @@ def main(args):
     parser.add_argument(
         "--overwrite", help="Whether to overwrite output files", action="store_true"
     )
+    parser.add_argument(
+        "--max_af",
+        help="Maximum variant AF to keep",
+        nargs="?",
```

Comment:
I don't think you need this
```suggestion
```

Reply:
It's used to make sure the input of `max_af` is a single optional argument, see https://docs.python.org/3/library/argparse.html#nargs

Reply:
Yeah, but I think that allows `max_af` to have nothing passed after it, and then it's just the same as not adding it. I think it's better to just remove.